File: coding-test/src/hye/프로그래머스/day241118/약수의_개수와_덧셈/solution.js
function solution() {
    var answer = 0;

    for(let j = left; j <= right; j++) {
        let count = 0;
        for(let i = 1; i <= Math.sqrt(j); i++){
            if(j % i == 0) {
                count++;
                if(i != j / i) {
                    count++;
                }
            }
        }
        if(count % 2 == 0) {
            answer += j;
        } else {
            answer -= j;
        }
    }


    return answer;
}

File: coding-test/src/hye/프로그래머스/day241118/약수의_개수와_덧셈/Solution.java
package hye.프로그래머스.day241118.약수의_개수와_덧셈;

class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        answer = s.matches("^[0-9]{4}$|^[0-9]{6}$");
        return answer;
    }
}


